about.html

```diff
@@ -12,15 +12,15 @@
     <div class="c_about">
         <p class="p1_about">Hi I'm a web developer, I'm from Ecuador, I'm 26 years old.
             I'm learning to make web pages and in here public a lot of things that I learn</p>
-        <img class="img_about"
+        <img class="img_about" role="img"
             src="https://scontent.fuio26-1.fna.fbcdn.net/v/t1.15752-9/294554831_573849710839999_3249083909119485489_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=ae9488&_nc_eui2=AeGVPloXPKjFXNfmn3lUNZUNLE28ezQ3gtssTbx7NDeC2zFGxzyOzTtKprS1tgXad20xByAMJ7PF3cDKyyo6u3Qk&_nc_ohc=YmxqtGV_998AX8DFgtQ&_nc_ht=scontent.fuio26-1.fna&oh=03_AVJc2GkQGHV4uQTN_MHVXKVJT50vwcdEF6ps7cHU9tVkVA&oe=630EEA91">
         <p class="p2_about">I am an environmental technician but In this moment I study web development, I loves this
             and and
             I hope to go very far in the future developing many web pages</p>
         <p class="p3_about">Stay tuned to my page and see my progress</p>
     </div>
-    <form action="index.html"> <a id="back"><input id="botton_SeeMore" type="submit" value="Back">
-            </input></a>
+    <form action="index.html"> <a id="back"><input id="botton_SeeMore" type="submit" value="Back" role="button">
+            </input></a><label for="botton_SeeMore"></label>
```

Comment:
A few things here:
1. Why do you have an anchor wrapping the button?
2. The label doesn't have a text and without it, it doesn't do anything
3. Buttons don't usually have labels, read about aria attributes and use one of those here